qm.te

```diff
@@ -30,7 +30,19 @@ unconfined_domain(ipc_t)
 
 qm_domain_template(qm)
 
+#########################################
+# IPC between QM containers
+#########################################
+
+require {
+    type qm_file_t;
+    type qm_container_ipc_t;
+    class sock_file write;
+}
```

Comment:
This `require` block can be removed, I think. 
Both types are defined in the qm policy - `qm_file_t` ([here](https://github.com/containers/qm/blob/main/qm.if#L52)) and `qm_container_ipc_t` ([here](https://github.com/containers/qm/blob/main/qm.if#L329)). And the `sock_file` class is being used further down by the stream_connect_pattern of the bluechi part of the policy.